src/lib/__tests__/moduleMocker-test.js

```diff
@@ -57,11 +57,11 @@ describe('moduleMocker', function() {
     it('does not mock non-enumerable getters', function() {
       var foo = Object.defineProperties({}, {
         nonEnumMethod: {
-          value: function() {}
+          value: function() {},
         },
         nonEnumGetter: {
-          get: function() { throw 1; }
-        }
+          get: function() { throw new Error(); },
```

Comment:
I know I know, in a previous comment I said I did not want to change any logic. But since this is a test and not part of the actual product I thought I makes more sense to change the code instead of disabling the lint warning.